steampipecloud/table_steampipecloud_workspace_pipeline.go

```diff
@@ -0,0 +1,614 @@
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "args",
+				Description: "Arguments to be passed to the pipeline.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "tags",
+				Description: "The tags for the pipeline.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "last_process_id",
+				Description: "The unique identifier of the last process that was executed for the pipeline.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromCamel(),
+			},
+			{
+				Name:        "last_process",
+				Description: "Information about the process that was last executed for the pipeline.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "query_where",
+				Description: "The query where expression to filter pipelines.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromQual("query_where"),
+			},
+			{
+				Name:        "created_at",
+				Description: "The time when the pipeline was created.",
+				Type:        proto.ColumnType_TIMESTAMP,
+			},
+			{
+				Name:        "created_by_id",
+				Description: "The unique identifier of the user who created the pipeline.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromCamel(),
+			},
+			{
+				Name:        "created_by",
+				Description: "Information about the user who created the pipeline.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "updated_at",
+				Description: "The time when the pipeline was last updated.",
+				Type:        proto.ColumnType_TIMESTAMP,
+			},
+			{
+				Name:        "updated_by_id",
+				Description: "The unique identifier of the user who last updated the pipeline.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromCamel(),
+			},
+			{
+				Name:        "updated_by",
+				Description: "Information about the user who last updated the pipeline.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "version_id",
+				Description: "The current version ID for the pipeline.",
+				Type:        proto.ColumnType_INT,
+				Transform:   transform.FromCamel(),
+			},
+		},
+	}
+}
+
+//// LIST FUNCTION
+
+func listWorkspacePipelines(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	var workspace *openapi.Workspace
+	switch w := h.Item.(type) {
+	case openapi.Workspace:
+		wo := h.Item.(openapi.Workspace)
+		workspace = &wo
+	case *openapi.Workspace:
+		workspace = h.Item.(*openapi.Workspace)
+	default:
+		plugin.Logger(ctx).Error("listWorkspacePipelines", "unknown response type for workspace list parent hydrate call", w)
+	}
+
+	// If the requested number of items is less than the paging max limit
+	// set the limit to that instead
+	maxResults := int32(100)
+	limit := d.QueryContext.Limit
+	if d.QueryContext.Limit != nil {
+		if *limit < int64(maxResults) {
+			if *limit < 1 {
+				maxResults = int32(1)
```

Comment:
We do not need to handle this check since the SDK will automatically throw an error for any value less than 1.